lib/commands/arm/hdinsight/hdinsight._js

```
@@ -1003,6 +1003,7 @@ hdInsightCommandLine.init = function(cli) {
     .option('-u, --uri <uri>', $('Script URI'))
     .option('-p, --parameters <parameters>', $('(Optional) Script parameters'))
     .option('-t, --nodeTypes <nodeTypes>', $('Specifies the nodes on which this cmdlet applies the action separated by ";". For example, "headnode;workernode" is a valid input.'))
+    .option('-a, --applicationName <applicationName>', $('Application name. When application name is specified, persistOnSuccess cannot be specified, and nodeTypes must contain only "edgenode"'))
```

Was the validation logic for --persistOnSuccess and --aplicationName to be mutually exclusive already there in the CLI as I do not see that in this PR?

It would not be ideal to send a request and let the server fail.